From 94676319b14ee43267ad79bebd106d91ac8688b4 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Wed, 25 Sep 2019 20:00:35 +0000
Subject: [PATCH] LuneOS: fix systemd service file

ls-hubd_private service is now simply called ls-hubd.
---
 LuneOS/systemd/sensorfwd.service | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/LuneOS/systemd/sensorfwd.service b/LuneOS/systemd/sensorfwd.service
index e480406..08caefe 100644
--- a/LuneOS/systemd/sensorfwd.service
+++ b/LuneOS/systemd/sensorfwd.service
@@ -1,7 +1,7 @@
 [Unit]
 Description=Sensor daemon for sensor framework
-After=dbus.socket ls-hubd_private.service
-Requires=dbus.service ls-hubd_private.service
+After=dbus.socket ls-hubd.service
+Requires=dbus.service ls-hubd.service
 Conflicts=actdead.target
 
 [Service]
-- 
2.17.0

